bin/cfg/Tomb1Main.json5

```diff
@@ -220,19 +220,19 @@
     // Enhances texture filtering at distances.
     "anisotropy_filter": 16.0,
 
-    // Lara walks to pickups and switches when nearby instead of teleporting
+    // Makes Lara walk to pickups and switches when nearby instead of teleporting to them.
     "walk_to_items" : false,
 
-    // How many save slots are available.
+    // Changes the number of available save slots.
     "maximum_save_slots": 25,
 
-    // Make Lara revert to pistols on new level start (inline with TombATI behaviour)
+    // Makes Lara revert to pistols at the start of each level (in line with TombATI behavior).
     "revert_to_pistols": false,
 
-    // Enhances savegames so graphic effects, waterfall mist, flame emitters, and
+    // Enhances savegames so that graphic effects, waterfall mist, flame emitters, and
     // more are saved instead of disappearing on load.
     "enable_enhanced_saves": true,
 
-    // Sound effects are randomly, slightly pitched to vary the game sounds.
+    // Allows sound effects to be randomly, slightly pitched to vary the game sounds.
```

Comment:
Do we need the comments in this file considering that it's designed to get written back to, and all the comments will get discarded?

Actually, do we even need to ship this file, considering that 
- the user is expected to use the config tool
- the game should do just fine without it, using the defaults for each setting
- the game will create the config file for the power users upon first launch

?

Reply:
I'd be in favour of removing the file altogether - it'd be one less place to have to define future settings. The only issue I can think of is if you install the game and launch the config tool before running the game - in this case it won't generate the file by default, you would have to open a file manually. But this could easily be changed so that it does generate it. Or the expectation could be that the user must launch the game once before running the config tool?

Reply:
I'd rather have the config tool work ootb without requiring the user to launch the game (and use good defaults).

Reply:
The default configuration file is now removed. The config tool will launch normally if it can't locate the default file, and it will have all properties initialised to their defaults (as defined in `specification.json` in the project resources). If you make a change in the tool and save, it will create the file. See [b27bc7c](https://github.com/rr-/Tomb1Main/pull/637/commits/b27bc7c9fb0f28786dbbb609a64194a4238523ff).

I had to make a small change in the game as without the default config file it wouldn't load. I believe the issue was that the screen resolution wasn't being set. When it couldn't find the config file, it was jumping to the `cleanup` label, so while the default settings were being loaded, it was skipping over the `g_AvailableResolutions` initialisation.
See [4ee0f5a](https://github.com/rr-/Tomb1Main/pull/637/commits/4ee0f5a3487067af292d756b533979a5b9ffd017).

I adjusted the log message here to become a warning rather than an error, and it will now pass `"{}"` to `Config_ReadFromJSON` instead of an empty string to avoid a further error message in that function. The warning message line breaches the 80 column rule so `make lint` shifted it over 3 lines. The wording can perhaps be amended here if this is not ideal.